examples/README.md

```diff
@@ -455,4 +455,29 @@ Device name is: AMD Radeon PRO W6800
 Running with single precision real data type:
 DFT example run_time dispatch
 DFT example ran OK
-```
\ No newline at end of file
+```
+
+```none
+$ LD_LIBRARY_PATH=lib ./bin/example_dft_real_fwd_usm 
+########################################################################
+# DFTI complex in-place forward transform with USM API example:
```

Comment:
`DFTI` looks like a hangover from old MKL.

Reply:
I always read it as DFT interface.

Reply:
Perhaps thats what it stands for. It appears in the closed-source MKL as -for example- `DFTI_INPLACE`. Maybe its fine if it stands for something sensible then.